BattleTank/Tanks/Tank.cs

```diff
@@ -446,10 +470,10 @@ public void Explode()
             if (alive)
             {
                 armor = 0;
-                if (((lives / 0.25f) % 4)==0)
+                if (((lives / 0.25f) % 4) == 0)
```

Comment:
Porównywanie liczb zmiennoprzecinkowych. Tutaj można zrobić na prawdę wiele ciekawych błędów, te porównania powinny wyglądać w ten sposób: ``Math.Abs((lives / 0.25) % 4) <= TOLERANCE``, gdzie tolerance w tym przypadku może być ``float.Epsilon``. Poniżej tolerance po prostu ``<=3``, ``<=2``. Ale to pozostawiam Tobie. Na przyszłość byłoby dobrze to poprawić.

Bo pojawia się pytanie, co się stanie kiedy wartość działania będzie np ``lives / 0.25 % 4`` będzie wynosić ``0.00004``?, albo ``2.32323``?

Reply:
👍